STAR-Delegation/draft-ietf-acme-star-delegation.md

```diff
@@ -1002,10 +1004,24 @@ Internet (MAMI). This support does not imply endorsement.
 
 - Initial version, some text extracted from draft-sheffer-acme-star-requests-02
 
-# CSR Template Schema
+# CSR Template: CDDL
+{: #csr-template-schema-cddl}
+
+Following is the normative definition of the CSR template, using CDDL {{RFC8610}}. The CSR template MUST be a valid JSON document, compliant with the syntax defined here.
+
+An additional constraint that is not expressed in CDDL but MUST be validated by the recipient is that all objects (e.g. `distinguishedName`) MUST NOT be empty when they are included, even when each separate property is optional.
+
+While the CSR template must follow the syntax defined here, neither the IdO nor
+the NDC are expected to validate it at run-time.
```

Comment:
I am not sure I understand what information this is expected to communicate.  ISTM we could just drop this sentence without loosing anything.  (One thing that confuses me is the idea of "validation at run-time".  What do you mean by that?)

Reply:
The idea is "the template syntax is simple enough and therefore the NDC does not need to incorporate a full scale JSON Schema or CDDL validator, in order to validate the received CSR template".

But I agree this is an implementation detail we can skip. Will remove.